app/src/main/java/org/listenbrainz/android/presentation/features/components/SimilarUserCard.kt

```diff
@@ -0,0 +1,138 @@
+package org.listenbrainz.android.presentation.features.components
+
+import android.content.res.Configuration.UI_MODE_NIGHT_NO
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.LinearProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.listenbrainz.android.R
+import org.listenbrainz.android.presentation.theme.ListenBrainzTheme
+import org.listenbrainz.android.presentation.theme.isUiModeIsDark
+import org.listenbrainz.android.presentation.theme.lb_purple
+import java.text.DecimalFormat
+
+@Preview(uiMode = UI_MODE_NIGHT_NO, showBackground = true)
```

Comment:
Can we show 2 previews like this?

Reply:
Also, can you move the previews below the view?

Reply:
Yes two previews at a time! Also will do.